### PR TITLE
335 block 8 chapter cards

### PR DESCRIPTION
**Pull request by sonaldhekale**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.page/
- After: https://335-block-8-chapter-cards--cn-website--netcentric.hlx.page/drafts/sdhekale/chapter-cards

### CODE REVIEW

**Comment by sonaldhekale** [comment]
@mihai-buricea-net or @tomasznetcentric could you please review this?

File: blocks/chapter-cards/chapter-cards.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    .chapter-cards-wrap {
+        grid-template-columns: 1fr 1fr 1fr;
+    }
+}
+
+.chapter-cards-item {
+    display: grid;
+    grid-template-areas: 'box';
+    background: var(--c-dark-plum);
+    height: 260px;
```
Comment (tomasznetcentric) [inline blocks/chapter-cards/chapter-cards.css:24]: 145px + 80px = 265px total height.

File: blocks/chapter-cards/chapter-cards.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+@media (min-width: 800px) {
+    .chapter-cards-wrap {
+        grid-template-columns: 1fr 1fr 1fr;
+    }
+}
+
+.chapter-cards-item {
+    display: grid;
```
Comment (tomasznetcentric) [inline blocks/chapter-cards/chapter-cards.css:21]: we don't need the display grid and the grid-template-area here.

File: blocks/chapter-cards/chapter-cards.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+.chapter-cards svg {
+   width: 38px;
+   height: 38px;
+   fill: currentcolor;
+}
+
+.icon-plus {
+    float: right;
+    cursor: pointer;
```
Comment (tomasznetcentric) [inline blocks/chapter-cards/chapter-cards.css:58]: since the svg icon is wrapped in a link it gets the cursor as a pointer.

**Comment by alexnerdrumhansen** [comment]
Tested and mostly looks fine.
Two comments however,
1. The specs mention Dark Teal to be #26EFE9, but it's currently ##06c7cc, is this expected? @Gonzalo-CG 
2. Can we add a Lock function to one of the cards to test that as well as https://github.com/Netcentric/cn-website/issues/335 mentions a locked card function. @sonaldhekale 
Thanks!
Screenshot 2023-08-04 at 11 34 03
Screenshot 2023-08-04 at 11 34 31

**Comment by tomasznetcentric** [comment]
Hey @alexnerdrumhansen :) what do you mean by **lock function** in your last comment?

**Comment by Gonzalo-CG** [comment]
@alexnerdrumhansen @sonaldhekale 

1. The color should be #26EFE9 as per specs.
2. The lock functionality was dropped after alignment w/ marketing as they don't see the need for it. The locking function was edited out of the specs.

**Comment by alexnerdrumhansen** [comment]
Thanks @Gonzalo-CG .
We should update the Main ticket too maybe just to document it there too that no lock mode is needed any more.
Only colour change needed for Dark Teal and we'll be good.
Thanks!

**Comment by alexnerdrumhansen** [comment]
Screenshot 2023-08-04 at 12 36 44
Looks fine, only we have different names for the colours in the Specs which causes some confusion when verifying.
https://projects.netcentric.biz/wiki/display/NCWEB/Block+8+Chapter+cards

**Pull request merged**